frontend/packages/otr/src/hooks/useAuthentication.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+import { APIResponseStatus } from 'shared/enums';
+
+import { useAppDispatch, useAppSelector } from 'configs/redux';
+import { AppRoutes } from 'enums/app';
+import { loadClerkUser } from 'redux/reducers/clerkUser';
+import { clerkUserSelector } from 'redux/selectors/clerkUser';
+
+export const useAuthentication = () => {
+  // Redux
+  const dispatch = useAppDispatch();
+  const clerkUser = useAppSelector(clerkUserSelector);
+
+  useEffect(() => {
+    const activeURL = window.location.href;
+    const clerkURL = AppRoutes.ClerkHomePage;
```

Comment:
Nää molemmat voisi varmaan kans vaan käyttää sellasenaan tuossa alempana ilman välillistä muuttujaa

Reply:
Ratkaisu näin AKR:ssä ja musta ihan fiksu. Muutin nyt tosin hieman tästä.